TeamCode/src/main/java/org/firstinspires/ftc/teamcode/MecanumDrive.java

```diff
@@ -440,18 +456,14 @@ public void preview(Canvas c) {
         }
     }
 
-    public PoseVelocity2d updatePoseEstimate() {
-        Twist2dDual<Time> twist = localizer.update();
-        pose = pose.plus(twist.value());
-
-        poseHistory.add(pose);
-        while (poseHistory.size() > 100) {
-            poseHistory.removeFirst();
-        }
-
-        estimatedPoseWriter.write(new PoseMessage(pose));
+    public Pose2d getLocalizerPose() {
+        return localizer.getPose();
+    }
```

Comment:
`localizer.getPose()` is about the same length as `getLocalizerPose()`... I prefer the former.